terraform/development/per_review_app/main.tf

```diff
@@ -81,6 +81,7 @@ module "application" {
   ecs_task_role_arn                                 = data.terraform_remote_state.development_shared.outputs.application_roles_ecs_task_role_arn
   export_bucket_details                             = data.terraform_remote_state.development_shared.outputs.cds_export_details
   govuk_notify_api_key_secret_arn                   = data.terraform_remote_state.development_shared.outputs.application_secrets_govuk_notify_api_key_secret_arn
+  openai_api_key_secret_arn                         = data.terraform_remote_state.development_shared.outputs.application_secrets_openai_api_key_secret_arn

```

Comment:
Can these be in alphabetical order please? (this probably applies every time this is added to a list in this PR)

Reply:
Ah, of course! Updated now!